micro_ros_diagnostic_updater/CMakeLists.txt

```diff
@@ -66,7 +74,9 @@ if(MICRO_ROS_DIAGNOSTIC_UPDATER_EXAMPLES)
 endif()
 
 # install
-install(DIRECTORY include/ DESTINATION include)
+install(DIRECTORY include/
```

Comment:
When I try to build, I get 

```
CMake Error at CMakeLists.txt:77 (install):
  install DIRECTORY does not allow "DESTINATION" after PATTERN or REGEX.
```

Reply:
Don't you think it'd be better to avoid regex and be explicit on what we install anyways? Makes it more clear to detect possible errors